specification/columns/listunitprice.md

```diff
@@ -20,10 +20,11 @@ The suggested provider-published unit price for a single Pricing Unit of the ass
 
 | Constraint      | Value                                |
 |:----------------|:-------------------------------------|
+| Column Type     | Metric                               |
```

Comment:
@salpaysenturus Should prices be dimensions or metrics?
```suggestion
| Column Type     | Dimension                            |
```

Reply:
Columns like this could be viewed either way, given that in a downstream semantic model, you might allow the user to both 1) slice with a single unit price, and 2) aggregate an average / min / max of many unit prices.

That said, we can't have Schrodinger's Column.  I'd call it a dimension.

Reply:
Sticking to metric for now.